Project 2/styles.css

```diff
@@ -0,0 +1,103 @@
+section {
+    background-color: #fff7e5;
+    max-width: 640px;
+    margin: 0 auto;
+    padding: 20px 30px;
+    border-radius: 80px;
+    border: solid rgb(255, 211, 226) 2px;
+
+}
+
+body {
```

Comment:
This ruleset should be right after your global box sizing rule, at the very top of your stylesheet, with all the other rules inheriting from it.